chrome/content/zotero/bibliography.js

```diff
@@ -87,10 +85,18 @@ var Zotero_File_Interface_Bibliography = new function() {
 			selectIndex = 0;
 		}
 		
+		// add locales to list
+		if(!_io.locale) {
+			_io.locale = Zotero.Prefs.get("export.lastLocale");
```

Comment:
I'm not sure it's a good idea to assign this to the `_io` object (the caller may not expect this value to be modified). Just do `Zotero.Styles.populateLocaleList(menulist, _io.locale || Zotero.Prefs.get("export.lastLocale"))` below